### DISCUSSION TITLE
Section numbers after applying a template

### DISCUSSION THREAD

**Discussion by zhengzhou0105** [davidgohel/officedown#106] [Q&A] [answered]
Hello community! How to enable section numbers after applying a template in which headers are unnumbered? 

Section numbers can be control using YAML option:
officedown::rdocx_document:
      reference_num: true
The value is true by default which enables section numbers. 

When a docx template is applied, the template seems to override the option above. Therefore, if the template has headers unnumbered, headers in the output will be unnumbered despite the value given to reference_num .

My question is, if I use a template with unnumbered headers, is it possible to re-enable section numbers?

* It seems to me that option loading order does not matter. Later options do not override earlier ones. 
* My goal is to make a word document with some headers unnumbered such as ABSTRACT and REFERENCE.

Any help is appreicated!

**Comment by alapo** [comment, marked as answer]
Have you tried wrapping the text you want using `custom-style`? I am going to be posting a video related to another question with line numbering (Issue #94)  shortly which may be of help to you. Its a walkthrough on how to apply a template.

**Reply by zhengzhou0105** [reply]
Thank you! I apologize that I forget to update this post. I found the solution on officedown website using custom-style. It is a flexible solution with lots of capacities.

I hope it could be implemented as the default style for docx-document. I would like to refer to book down::word_document2 that section numbers are automatically included in caption. Furthermore, an argument of global_numbering could be specified to toggle off section numbers.
